.github/workflows/pipeline.yml

```diff
@@ -25,10 +25,6 @@ jobs:
     uses: ./.github/workflows/windows.yml
   macos:
     uses: ./.github/workflows/macos.yml
-  release_test:
```

Comment:
This was more necessary before when releases were once a week. But now this runs much of the whole steps twice on every run. When uploads should already be gated by the equivalent of when dry-run was false. So just dropping completely.